Formula/crystal-lang.rb

```diff
@@ -1,31 +1,10 @@
 class CrystalLang < Formula
   desc "Fast and statically typed, compiled language with Ruby-like syntax"
   homepage "https://crystal-lang.org/"
-  revision 1
-
+  url "https://github.com/crystal-lang/crystal/archive/0.19.0.tar.gz"
+  sha256 "4d7d9770891bd8bd835251e2654316412a3f44074db0adeca28357c0993eb2d8"
   head "https://github.com/crystal-lang/crystal.git"
-
-  stable do
-    url "https://github.com/crystal-lang/crystal/archive/0.18.7.tar.gz"
-    sha256 "72b33fd4bd903a9f0957c74c2f95429e8c0f94c04df86396711b5560f540805d"
-
-    # LLVM 3.8 support patches begin.
-    patch do
-      url "https://github.com/crystal-lang/crystal/commit/53f1977c8c.patch"
-      sha256 "a89ba25a5e7469c225680a58e87d4342a2709477dce61215803d9422cf8d0860"
-    end
-
-    patch do
-      url "https://github.com/crystal-lang/crystal/commit/19ddb4c3a.patch"
-      sha256 "f98b92d9003d2fa69fe36e218329cc616198e4a6283ba81b7c27e84d9d4afac0"
-    end
-
-    patch do
-      url "https://github.com/crystal-lang/crystal/commit/63ca4950e4.patch"
-      sha256 "bb0cbb466673f7f04996326c875c8c7f9a0335e31a3bfab1576358fdf8a697f1"
-    end
-    # LLVM 3.8 support patches end.
-  end
+  revision 1
```

Comment:
Ah, OK.

Then why this PR was accepted, the one that introduced that revision? https://github.com/Homebrew/homebrew-core/commit/f1d68e0bf53e3be40be86575415652153c7e6862

Is there anything I can do so that only me can send PR to this formula?